core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -983,13 +982,8 @@ public List<Team> selectTeamsOfUsers(String username, int tenantId) {
 
     Integer teamId = userInfoOpt.get().getTeamId();
 
-    Team teamSel = allTeams.get(teamId);
-
-    if (teamSel == null) {
-      return Collections.emptyList();
-    }
-
-    return List.of(teamSel);
+    log.debug("selectTeamsOfUsers {}", username);
+    return selectAllTeamsByTenantIdAndTeamId(tenantId, teamId);
```

Comment:
@snuyanzin what is returned in the scenario where the select all finds no matching team in the tenant?
(It shouldnt happen but just curious will it return an empty list, I think it is which would make this change perfect but just wanted to double check)

Reply:
in javadocs for `findAllXXX` itis mentioned that
>Returns:
guaranteed to be not null. The size can be equal or less than the number of given ids.

so yes in case of nothing to matched there should be an empty collection